gateway/src/test/java/io/zeebe/gateway/security/SecurityTest.java

```diff
@@ -38,6 +38,7 @@ public void shouldNotStartWithTlsEnabledAndWrongCert() throws IOException {
     cfg.getSecurity()
         .setCertificateChainPath(
             cfg.getSecurity().getCertificateChainPath().replaceAll("test-chain", "wrong-chain"));
+    cfg.init();
```

Comment:
why we need this?

Reply:
Good point. We don't need to call `init()`.